app/src/routes/onboarding/forms/project.ts

```diff
@@ -0,0 +1,249 @@
+import { Field } from '@directus/types';
+import { useI18n } from 'vue-i18n';
+
+export const useProjectFields = function (): Field[] {
+	const { t } = useI18n();
+	return [
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_settings.project_name'),
+			field: 'project_name',
+			type: 'string',
+			schema: {
```

Comment:
I think we should clean-up the schema (same in `app/src/routes/onboarding/forms/user.ts`), removing necessary properties, especially the `id`s

Reply:
Wondering the same thing as well. Maybe using `DeepPartial<Field>[]` instead of `Field[]` will solve it (assuming ids etc were added to conform to `Field` type). Similar to approaches such as:

https://github.com/directus/directus/blob/fea4cc4e42f260c3ecc80ff7e9e94c77aeb388b2/app/src/interfaces/list/options.vue#L39

https://github.com/directus/directus/blob/0ee6e198020c27eb07f6082aabefbf5b45af3f9d/app/src/modules/settings/routes/data-model/new-collection.vue#L186